### PR TITLE
New Update

### PR DESCRIPTION
**Pull request by GeoffreyWesthoff**
- Added profile image generation
- Added altstyle ("Twitter-esque") style to meme generation
- Added text-endpoint "yomomma"
- Other tweaks and fixes

### CODE REVIEW

**Review by Nevvulo — APPROVED** [review]

File: endpoints/yomomma.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+@setup
+class YoMomma(Endpoint):
+    """
+    This endpoint only returns a yo momma joke. No parameters are required.
+    """
+    params = []
+
+    def generate(self, avatars, text, usernames, kwargs):
+        choices = ['Yo momma so fat when she walked past the TV I missed three episodes',
```
Comment (devoxin) [inline endpoints/yomomma.py:15]: Perhaps these should be exported to a txt file split by new line to keep the endpoint looking clean(er)?

File: server.py [hunk 2]
```diff
@@ -133,6 +133,11 @@ def api(endpoint):
                            'X-RateLimit-Reset': e_r['X-RateLimit-Reset'],
                            'Retry-After': e_r['Retry-After']}))
         return x
+    if endpoint == 'profile':
+        if request.headers.get('Authorization', None) == config.get('memer_token', None):
+            pass
```
Comment (devoxin) [inline server.py]:
```python
[...] != [...]:
  return 'blah'
```
removes the need for `pass/else`

**Review by devoxin — CHANGES_REQUESTED** [review]

File: server.py (hunk 2 above)
Comment (GeoffreyWesthoff) [reply server.py]: oh yeah

File: endpoints/yomomma.py (hunk 1 above)
Comment (GeoffreyWesthoff) [reply endpoints/yomomma.py:15]: Yeah might be better, will do that later

**Pull request merged**